Source/WebCore/crypto/algorithms/CryptoAlgorithmX25519.cpp

```diff
@@ -87,12 +87,20 @@ void CryptoAlgorithmX25519::deriveBits(const CryptoAlgorithmParameters& paramete
         return;
     }
 
+    // Return an empty string doesn't make much sense, but truncating either at all.
+    // https://github.com/WICG/webcrypto-secure-curves/pull/29
+    if (length && !(*length)) {
+        // Avoid executing the key-derivation, since we are going to return an empty string.
+        callback(Vector<uint8_t>());
```

Comment:
Assuming there is no style error in this.
```suggestion
        callback( { } );
```

Reply:
Done